SDK/CalendarStoreViewController.swift

```diff
@@ -175,10 +179,27 @@ public final class CalendarStoreViewController: UITabBarController {
             if #available(iOS 11.0, *) {
                 navigationController.navigationBar.prefersLargeTitles = largeTitle
             }
+            
+            if showCloseButton == true &&
```

Comment:
`showCloseButton == true`? Can't that just be `showCloseButton`?

Reply:
Yes, but I prefer this style, it's clearer for me at first glance.